### DISCUSSION TITLE
Nutror

### DISCUSSION THREAD

**Discussion by focandonofisco** [aclap-dev/video-downloadhelper#486] [Help and Q&A] [answered]
### Your question

Hi everyone,
I'm trying to download NUTROR courses, but isn´t working anymore and I get the Error: Exit code: 1
Does anyone have any idea what might be going on?

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: pt-PT
license: accepted
key (only 16 first characters): 6a2cde0e270f42b1
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\conta",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
You need to do what I explained in #485.

**Comment by gilmirabelli** [comment, marked as answer]
Download and install version 7.6.6 and it will work. You will have to reinstall it when updated. But it's a good work around.
https://addons.mozilla.org/firefox/downloads/file/4040817/video_downloadhelper-7.6.6.xpi

**Reply by focandonofisco** [reply]
Thank you, it worked!

**Reply by gilmirabelli** [reply]
Bom proveito.
